cluster/gce/manifests/etcd.manifest

```diff
@@ -15,7 +15,7 @@
 "containers":[
     {
     "name": "etcd-container",
-    "image": "{{ pillar.get('etcd_docker_repository', 'us.gcr.io/k8s-artifacts-prod/etcd') }}:{{ pillar.get('etcd_docker_tag', '3.4.7-2') }}",
+    "image": "{{ pillar.get('etcd_docker_repository', 'us.gcr.io/k8s-artifacts-prod/etcd') }}:{{ pillar.get('etcd_docker_tag', '3.4.9-1') }}",
```

Comment:
this needs to switch back to the old prod registry?

Reply:
Thanks. But both new and the old prod registry have image for this version.